tests/BasicTestBaseline/BasicTestBaseline.csproj

```diff
@@ -6,7 +6,7 @@
     <AssemblyName>BasicTestBaseline</AssemblyName>
     <OutputType>Exe</OutputType>
     <PackageId>BasicTestBaseline</PackageId>
-    <RuntimeIdentifiers>win7-x64</RuntimeIdentifiers>
+    <!--<RuntimeIdentifiers>win7-x64</RuntimeIdentifiers>-->
```

Comment:
Uncomment or remove?

Reply:
it isn't the core - I'm fine with leaving it for context